### DISCUSSION TITLE
Custom partition xavier nx

### DISCUSSION THREAD

**Discussion by yrodriguez31** [OE4T/meta-tegra#1546] [Q&A]
Hello i try to create a custom partition for my xavier-nx board.
I start with creating a custom machine like that :
```
require conf/machine/jetson-xavier-nx-devkit.conf
MACHINEOVERRIDES = "cuda:tegra:tegra194:xavier-nx:${MACHINE}"
PACKAGE_EXTRA_ARCHS:append = " jetson-xavier-nx-devkit"
#PARTITION_LAYOUT_TEMPLATE = "flash_orphie.xml"

```
I comment the partition layout for now because that not working when i try to flash .
When i flash using the meta-tegra script i have the folowing error:

ERR: flash variable set not defined

Do you have an idea on what i'm missing . I follow the wiki and the examples . But i can't get over this error.

Thanks

**Comment by madisongh** [comment]
Sounds like you have an empty `flashvars` file, which means you either overlooked, or made a mistake making the modifications mentioned on the wiki page for customizing the `tegra-flashvars` recipe for your custom machine.

**Reply by yrodriguez31** [reply]
Thanks for your help
